titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/sanitizer/JobConfiguration.java

```diff
@@ -80,7 +80,7 @@
      * An upper bound on disk (megabytes) a single container may allocate. The actual limit may be lower, as it also depends
      * on instance types available in a tier.
      */
-    @DefaultValue("999000")
+    @DefaultValue("1550000")
```

Comment:
(just a reminder that the Docker image size is counted in this budget, which is why we currently set the min at 10GB)

Reply:
yes